inat_finder.py

```
@@ -0,0 +1,589 @@
+    observation_number_or_url  The potentially mistyped iNaturalist observation number
+                               or a complete iNaturalist URL
+
+Options:
+    --digits N          Number of digits that might be wrong (default: 1)
+    --verbose           Print detailed information about each attempt
+    --no-progress       Hide the progress bar (progress bar is shown by default)
+"""
+
+import argparse
+import requests
+import time
```

Pylint (3.3.7)

[convention] standard import "time" should be placed before third party import "requests"

(C0411)